src/page-objects/administration/Categories.ts

```diff
@@ -0,0 +1,25 @@
+import type { Page, Locator } from '@playwright/test';
+import type { PageObject } from '../../types/PageObject';
+
+export class Categories implements PageObject {
+
+    /**
+     * Landing pages
+     */
+    public readonly landingPageArea: Locator;
+    public readonly landingPageHeadline: Locator;
+    public readonly addLandingPageButton: Locator;
+    public readonly landingPageItems: Locator;
+
+    constructor(public readonly page: Page) {
+
+        this.landingPageArea = page.locator('.sw-category-detail__landing-page-collapse');
+        this.landingPageHeadline = this.landingPageArea.locator('.sw-category-detail__collapse-headline');
```

Comment:
here the build in locator:

page.getByRole('heading', { name: 'Landing Page' });